webpack.config.js

```diff
@@ -31,5 +30,10 @@ module.exports = {
         }
       }
     ]
-  }
-}
+  },
+  plugins: [
+    new DefinePlugin({
+      'process.env.NODE_ENV': argv.mode
```

Comment:
`argv.mode || 'development'`? Maybe it already has a default value though!

Reply:
https://webpack.js.org/concepts/#mode - default is "production".